src/components/HelpText/styled/index.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+
+export const StyledButton = styled.button`
```

Comment:
I think we should use our `Button` component,
@reiniergs and @LeandroTorresSicilia whats do you think?